downloads.html

```diff
@@ -247,17 +247,20 @@ <h3>Importing keys and verifying builds</h3>
       var blobs = [];
       var retrieveBlobs = function(marker, finished) {
         // Generate the blob listing URL and request it from Azure
-        var url = 'https://gethstore.blob.core.windows.net/builds?restype=container&comp=list&maxresults=5000&prefix=geth-';
+        var url = 'https://gethstore.blob.core.windows.net/builds?restype=container&comp=list&maxresults=250&prefix=geth-';
         if (marker != "") {
           url += "&marker=" + marker;
         }
         $.ajax({
           url: url,
           type: 'GET',
+          dataType: 'xml',
+          timeout: 3 * 60 * 1000,
           error: function() {
             alert("Failed to load releases!");
+            // Remove the modal blocker, so the user can at least access the things which have loaded
+            $('#loader').modal('hide');
```

Comment:
This will still block for 3 minutes, which is not very nice either. IMO it would be nicer to just adda a close-button on the modal so the user can click to close it. I suspect it should already exist in the jquery standard toolkit.

Reply:
I think this is just a `div` with `modal` set, we could just a proper dialog instead: https://jqueryui.com/dialog/#default . Then it would by default be closable by the user.

Reply:
I tested the code above in my browser console.

Unfortunately I cannot make the larger change you suggest without properly testing.

And I cannot test right now, because `docker-compose up` isn't working for me!

```
go-ethereum-jekyll-1  | /usr/local/lib/ruby/site_ruby/2.7.0/rubygems.rb:281:in `find_spec_for_exe': Could not find 'bundler' (1.17.2) required by your /srv/jekyll/Gemfile.lock. (Gem::GemNotFoundException)
```

So perhaps we should proceed with this change, and leave the ideal change for a later update.

Yes this will still block for 3 minutes, which isn't ideal. But it's better than the current behaviour, which is blocking forever!